src/jobs/check-health.yml

```diff
@@ -0,0 +1,48 @@
+description: >
+  This job checks the specified AWS region for any issues. The job passes if there are no issues found.
+  If there are issues in the region, the job will continue to check for issues every 10 seconds until
+  no issues are detected. If the max attempts have been reached, the job will fail.
+
+executor: <<parameters.executor>>
+
+parameters:
+  executor:
+    type: executor
+  aws-region-to-check:
+    type: env_var_name
+    default: AWS_REGION
+    description: >
+      The AWS region to be checked for issues. It defaults to AWS_REGION that's set as an environment variable
+  max-poll-attempts:
+    type: integer
+    default: 6
+    description: >
+      The max number of attempts to recheck for issues in the specified AWS region. Each attempt takes
+      10 seconds. By default, 6 attempts will be made for a total of 1 minute.
+  aws-access-key-id:
+    type: env_var_name
+    description: aws access key id override
+    default: AWS_ACCESS_KEY_ID
+  aws-secret-access-key:
+    type: env_var_name
+    description: aws secret access key override
+    default: AWS_SECRET_ACCESS_KEY
+  aws-region:
+    type: env_var_name
+    description: aws region override
+    default: AWS_REGION
+  profile-name:
+    description: AWS profile name to be configured.
+    type: string
+    default: ''
+  install-aws-cli:
+    description: The aws-cli is installed when this parameter is set to true. Set to false to skip installation
+    type: boolean
+    default: true
+
+steps:
+  - check-health:
```

Comment:
The following parameters are defined in the job but never explicitly used in it or passed to the command:

- aws-region-to-check
- max-poll-attempts
- install-aws-cli

Do you plan to add them later on?

Reply:
Good call, I've added them